src/cpp/desktop/DesktopMain.cpp

```diff
@@ -639,6 +643,15 @@ int main(int argc, char* argv[])
          arguments.push_back(noSandbox);
       }
 
+      static char disableSeccompFilterSandbox[] = "--disable-seccomp-filter-sandbox";
+
+      // newer versions of glibc require us to disable the seccomp filter sandbox,
```

Comment:
Feels like we'll want to mention this (in NEWS for sure, but maybe even a support article?). People will want to know the implications of running with the sandbox disabled.